.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+default_language_version:
+    python: python3.12
+repos:
```

Comment:
what do you think about using also
isort
bandit
:) ?

Reply:
uu i think that's an excellent idea! I added both of them to my project :)